currencies.json

```diff
@@ -264,6 +264,32 @@
         },
         "format": "-CA$###,###,###.##"
     },
+    "CNY": {
+        "name": "Chinese Yuan",
+        "iso": {
+            "code": "CNY",
+            "num": "156"
+        },
+        "symbol": {
+            "default": {
+                "display": "¥",
+                "code": "&#xa5;"
+            },
+            "native": {
+                "display": "元",
+                "code": "&#x5143;"
+            }
+        },
+        "units": {
+            "decimal": 2,
+            "sub": 100,
+            "name": {
+                "major": "Yuan",
+                "minor": "Fen"
```

Comment:
china actually has 2 different minors: jiao (10 jiao = 1 yuan) & fen (10 fen = 1 jiao).
i'm not sure which one to use, actually. so i picked fen, since "cent" is more popularly used than "dime" in US (i think?)
however, [XE says the minor is jiao](https://www.xe.com/currency/cny-chinese-yuan-renminbi/).
so, lemme know if you want me to change it